python/ray/serve/logging_utils.py

```diff
@@ -0,0 +1,82 @@
+import logging
+import os
+from typing import Optional
+
+import ray
+from ray.serve.constants import DEBUG_LOG_ENV_VAR, SERVE_LOGGER_NAME
+
+COMPONENT_LOG_FMT = "%(levelname)s %(asctime)s {component_name} {component_id} %(filename)s:%(lineno)d - %(message)s"  # noqa:E501
+LOG_FILE_FMT = "{component_name}_{component_id}.log"
+
+
+def access_log_msg(*, method: str, route: str, status: str, latency_ms: float):
+    """Returns a formatted message for an HTTP or ServeHandle access log."""
+    return f"{method.upper()} {route} {status.upper()} {latency_ms:.1f}ms"
+
+
+def configure_component_logger(
+    *,
+    component_name: str,
+    component_id: str,
+    component_type: Optional[str] = None,
+    log_level: Optional[int] = logging.INFO,
```

Comment:
nit, the type isn't really optional, optional mean it can literally be None

Reply:
oops, thanks